collections/business/general-notifications/event-general-notifications/email-address-added/email-address-added/email/assets/email-address-added-template.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+<div>
+  <p>Dear {{{ user.preferredName }}},</p>
+  <br />
+  <p>A new email address was added to your online banking on {{ formatDate event.dateTime 'MMMM d yyyy \'at\' HH:mm:ss z' 'en' 'US/Eastern' }}.</p>
+  <p>If this was you:</p>
+  <br />
+  <ul>
+    <li>You don't have to do anything</li>
+  </ul>
+  <p>If this wasn't you:</p>
+  <ul>
+    <li>Contact us right away. You should also:</li>
+    <ul>
+      <li>Change your password, and check your online banking for activity.</li>
+      <li>Make sure your phone numbers on online banking are correct.</li>
```

Comment:
That's text correct?
According to https://backbase.atlassian.net/wiki/spaces/ID/pages/4463428258/Notification+Copy+and+Primary+Secondary+Email+Logic should be: Remove the email address from your online banking